src/Helpers/Filters/sort.jsx

```diff
@@ -0,0 +1,5 @@
+// const getSortedProducts =  (products, sortBy) =>{
```

Comment:
Suggestion: if not needed you can remove file

Reply:
ok